lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -638,3 +643,46 @@ static void start_stats_timer(void) {
       &s1ap_task_zmq_ctx, 1000 * epc_stats_timer_sec, TIMER_REPEAT_FOREVER,
       handle_stats_timer, NULL);
 }
+
+static void start_state_hashtable_timer(void) {
+  state_hashtable_timer_id = start_timer(
+      &s1ap_task_zmq_ctx, 1000 * state_hashtable_timer_sec,
+      TIMER_REPEAT_FOREVER, handle_state_hashtable_timer, NULL);
+}
+
+static int handle_state_hashtable_timer(zloop_t* loop, int id, void* arg) {
```

Comment:
we simply want to track the number of elements. why do we fetch all the keys and values?

Reply:
updated all timer handlers. I was verifying if there were any hanging keys without value structures, but from the experiments being run they all returned 0 stale entries.